src/style.css

```diff
@@ -1 +1,37 @@
 /* add styles here */
+
+.page__body {
+  margin: 0;
+}
+
+.stars {
+  justify-content: align-items;
+}
+
+/* #endregion */
+
+/* #region gray stars */
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
```

Comment:
In line with the checklist, it's recommended to be consistent with your margins. It's better to add margins to only one side (top or bottom) to avoid potential margin collapse. Here you've applied a right margin.